srcpkgs/PolyMC/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'PolyMC'
+pkgname=PolyMC
+version=1.0.6
+revision=1
+_nbtversion="dc72a20b7efd304d12af2025223fad07b4b78464"
+_quazipversion="c9ef32de19bceb58d236f5c22382698deaec69fd"
+build_style=cmake
+configure_args="-DLauncher_BUILD_PLATFORM=Void -DLauncher_LAYOUT=lin-system"
+hostmakedepends="openjdk11 qt5-host-tools qt5-qmake"
+makedepends="qt5-devel"
+depends="virtual?java-environment"
```

Comment:
Surely we don't need a `java-environment` (JDK) but just a `java-runtime` (JRE)?

Reply:
This makes sense, now that jre and jdk are split again with 17.

Reply:
either way, it should be runtime because you don't need the jdk to run it, you need the runtime. jdk11 doesn't split it into a subpkg for whatever reason but it still provides runtime and environment